src/midiEditorCommands.cpp

```diff
@@ -1485,18 +1485,18 @@ string describeCC(MidiControlChange cc, MediaItem_Take* take) {
 			"note"_a=getMidiNoteName(take, cc.message2, cc.channel),
 			"value"_a=cc.message3);
 	}
-	if (cc.message1 == 0xB0) {
+	if (cc.message1 == 0xB0 ) {
```

Comment:
nit: Extraneous white space.

```suggestion
	if (cc.message1 == 0xB0) {
```